docker-compose.spark.yml

```diff
@@ -0,0 +1,56 @@
+version: '2'
+services:
+  spark-master:
+    image: spark-master
+    build:
+      context: ./worker-tasks
+      dockerfile: Dockerfile
+    image: quay.io/azavea/spark:1.6.2
+    restart: always
+    ports:
+      - "7077:7077"
+      - "8888:8080"
+    environment:
+      SPARK_LOCAL_DIRS: "/tmp"
+      SPARK_DAEMON_MEMORY: "256m"
+      SPARK_DAEMON_JAVA_OPTS: "-Dspark.deploy.recoveryMode=FILESYSTEM -Dspark.deploy.recoveryDirectory=/spark-state"
+    volumes:
+      # Spark cluster state
+      - /spark-state
+      # Spark scratch space
+      - /tmp
+    entrypoint: spark-class
+    command: org.apache.spark.deploy.master.Master
+
+  spark-worker:
```

Comment:
The main reason I didn't do this is because you can scale the workers, and when you do that it causes a conflict.

Regardless, I overrode and exposed the worker UI in https://github.com/azavea/raster-foundry/pull/424/commits/c9e6df121c13a091b5df52a99838c44b9dba9389.